src/3d/qgstiledscenechunkloader_p.cpp

```diff
@@ -87,7 +87,7 @@ QgsTiledSceneChunkLoader::QgsTiledSceneChunkLoader( QgsChunkNode *node, const Qg
     entityTransform.ecefToTargetCrs = &mFactory.mBoundsTransform;
 
     QStringList errors;
-    mEntity = QgsGltf3DUtils::gltfToEntity( gltfData, entityTransform, uri, &errors );
+    mEntity = QgsGltf3DUtils::gltfToEntity( tileContent.gltf, entityTransform, uri, &errors, tileContent.rtcCenter );
```

Comment:
How do you feel about passing rtc center within `QgsGltf3DUtils::EntityTransform::tileTransform` instead of having an extra parameter in gltfToEntity()?

Reply:
Ok, done!